### PR TITLE
adds the updates inside the service.maintenances object

### PR DESCRIPTION
**Pull request by ybriffa**
Adds the missing object updates for the maintenances to list the current maintenances of the service

### CODE REVIEW

**Review by Serpentiel — COMMENTED** [review]
please address the comment to the PR

**Comment by ybriffa** [comment]
> hey, @ybriffa 👋
> 
> thank you for your contribution!
> 
> would you be able to point me out where exactly in the API docs is the described addition, as I was unable to see it, at least for the `ServiceCreate` method that depends on it in the Go client:
> 
> image
> 
> thank you!

Hello,

Here we are: https://api.aiven.io/doc/#tag/Service/operation/ServiceList

This information is only available in read method I think

File: service.go [hunk 1]
```diff
@@ -168,8 +168,17 @@ type (
...
-		DayOfWeek string `json:"dow"`
-		TimeOfDay string `json:"time"`
+		DayOfWeek string               `json:"dow"`
+		TimeOfDay string               `json:"time"`
+		Updates   []*MaintenanceUpdate `json:"updates,omitempty"`
+	}
+
+	// MaintenanceUpdate represents a maintenance needing to be applied on the service.
+	MaintenanceUpdate struct {
+		Deadline    *string `json:"deadline"`
```
Comment (Serpentiel) [inline service.go:178]: please mark all the properties of this struct as `omitempty`

**Review by Serpentiel — CHANGES_REQUESTED** [review]

File: service.go (hunk 1 above)
Comment (ybriffa) [reply service.go]: it's done :)

**Comment by ybriffa** [comment]
Can we proceed on this one ?

**Review by Serpentiel — APPROVED** [review]
lgtm

**Comment by Serpentiel** [comment]
> Can we proceed on this one ?

sorry, we had a public holiday in Finland yesterday

**Pull request merged**